### ISSUE TITLE
Add in 'Alignment'

### ISSUE THREAD

**Issue by nvs** [nvs/gem#209]
Basically, an alternative to 'Extra Chance'. Rather than spending gold to give a one-off chance for a single round to get a particular gem, permanently augment the chances. For example, say `+1.20%`, `+0.50%`, and `+0.30%` to three particular gem types that are the focus of a particular alignment, and then `-0.50%` to those other five gem types.

The gold cost would increase each investment in a particular alignment, so spreading out your choices would overall be cheaper, but may not influence your luck quite as drastically as dumping into a single alignment.

Some numbers, using the above example and ten investments into a single alignment:

```
24.5% Primary Type
17.5% Secondary Type
15.5% Tertiary Type
8.5% Other Types (x5)
```

**Comment by nvs** [comment]
Blocked by a number of issues:
- General board rework/customizations #114 
- Gem placement/selection rework #168

**Comment by nvs** [comment]
Due to the plan to touch upon slate extra chance  in #242, the impetus for Alignment has lessened. In fact, I'm not even sure if it will be implemented. At least, not right away. The concept of a permanent way to augment type chance is interesting, as the ability exists for quality already. But, for now, this concept will be put on hold pending further discussion.
